### PR TITLE
Feature: Login Page

### PR DESCRIPTION
**Pull request by Cireimu**

### CODE REVIEW

**Comment by imriven** [comment]
Thank you for fixing my error! Still not too sure what happened.

**Review by imriven — APPROVED** [review]
Thanks for taking a look at this and getting the cp env test to pass!

**Comment by imriven** [comment]
[Component: Login](https://trello.com/c/rMg8SZcz/33-component-login)

File: src/components/pages/Login/style.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+padding: 0;
+display:flex;
+width: 99vw;
+height: 98vh;
+justify-content: center;
+align-items: center;
+align-content: center;
+overflow-y: hidden;
+overflow-x: hidden;
+@media (max-width: 320px) {
```
Comment (vtopham) [inline src/components/pages/Login/style.js:19]: nice work getting ahead on the responsiveness

**Review by vtopham — APPROVED** [review]
Love the code, glad to see that you're looking ahead with regards to responsiveness!

**Pull request merged**